saleor/core/models.py

```diff
@@ -189,6 +189,12 @@ def save_payload_file(self, payload_data: str):
         file_path = safe_join(prefix, file_name)
         self.payload_file.save(file_path, ContentFile(payload_bytes))
 
+    def save_as_file(self):
+        payload_data = self.payload
+        self.payload = ""
+        self.save()
```

Comment:
Should it be like so instead? Not sure if the full save is intended or not?

```suggestion
        if self.payload != "":
            self.payload = ""
            self.save(update_fields=["payload"])
```

Reply:
Full save is here on purpose as this methos should work also for unsaved objects
https://github.com/saleor/saleor/blob/b3f52b066aa682a8aad9508cfd33c4610e73fd84/saleor/webhook/transport/utils.py#L466